db/dcrpg/internal/stakestmts.go

```diff
@@ -18,7 +18,10 @@ const (
 		spend_type INT2,
 		pool_status INT2,
 		spend_height INT4,
-		spend_tx_db_id INT8
+		spend_tx_db_id INT8,
+		block_time INT8,
+		agenda_id TEXT,
+		agenda_vote_choice TEXT
```

Comment:
Why are agenda_id and agenda_vote_choice in the tickets table?  This would make more sense in the votes table I believe.

Reply:
This was a mistake and it has been corrected